code/datums/components/tackle.dm

```diff
@@ -309,6 +309,13 @@
 	if(HAS_TRAIT(sacker, TRAIT_GIANT))
 		attack_mod += 2
 
+	if(ismoth(sacker))
+		var/obj/item/organ/external/wings/moth/sacker_wing = sacker.getorganslot(ORGAN_SLOT_EXTERNAL_WINGS)
+		if(!sacker_wing || sacker_wing.burnt) // moths without wing/burnt ones can't tackle properly
+			attack_mod -= 2
+		else
+			attack_mod += 2 // healty wings are used as extra propulsion
+
```

Comment:
Hardcoding species checks is bad. Do not do it

```suggestion
if(locate(/obj/item/organ/external/wings/moth) in sacker.dna?.species.external_organs)
  var/obj/item/organ/external/wings/moth/sacker_wing = sacker.getorganslot(ORGAN_SLOT_EXTERNAL_WINGS)
  if(!sacker_wing || sacker_wing.burnt) // moths without wing/burnt ones can't tackle properly
	  attack_mod -= 2
  else
	  attack_mod += 2 // healty wings are used as extra propulsion

```

Reply:
I will make it into traits for each species.
It is easier for someone checking a species code to know they are better at tackling without having to dig into the component code.